docs/content/management-decision-logs.md

```diff
@@ -253,6 +253,36 @@ to track **remove** vs **upsert** mask operations.
 }
 ```
 
+### Dropping decisions
```

Comment:
Lets stick to case standard used in the docs and perhaps rephrase to "Drop Decision Logs"